### ISSUE TITLE
ucr_lstm_test

### ISSUE THREAD

**Issue by IlyaChernoglazov** [nla-group/fABBA#7]
I can't find the symbols_rnn method in src although I have executed setup.py. How can I repeat the experiment ?

**Comment by chenxinye** [comment]
@IlyaChernoglazov  
Apologize for the late reply. What symbols_rnn method? I didn't find it, sorry.

**Comment by IlyaChernoglazov** [comment]
Hi, https://github.com/nla-group/fABBA/blob/master/exp/ucr_lstm_test.ipynb from src import symbols_rnn, but src doesnt include this method

**Comment by chenxinye** [comment]
Yes, the repo is an old repo. After the paper of ABBA-LSTM was made public, fABBA was developed later. We will consider to include it in future. 

I suggest you might have a look at other repo https://github.com/inEXASCALE/llm-abba which currently under maintenance.
